readme.md

```diff
@@ -1,22 +1,121 @@
 <img src="https://forum.saturn.network/uploads/default/original/2X/e/e87ea6b5fb70b6044373d83cc89eb2d8a6c86449.png">
 
 # Saturn.py
-Port of Saturn.js to Python.
-https://forum.saturn.network/t/saturn-py-source/
+Version 0.0.2 by WizardsOrb and PseudoDeterminist, Oct 2019
+This version includes the well-formed-trades version of Saturnpy. INVALID TRADES WILL NOT BE SENT to the exchange contract.
 
-# NOTE
-```
-SATURN.PY IS ALREADY WORKING!
-DECIMAL PRECISION IS NOT PERFECT, FEEL FREE TO CONTRIBUTE AND FIX PRECISION.
-PLEASE USE AT YOUR OWN RISK WITH SMALL TRADING AMOUNT!
-```
+* https://github.com/WizardsOrb/saturn.py
+* https://forum.saturn.network/t/saturn-py-source
 
 
-# Depencies
-* python3+
+### Optional Settings
+The following flag values default to False, to allow the simplest and best trading experience for most users.
+* `SBTKN` (default: `False`) means TRADES ARE REQUESTED IN DECIMAL TOKEN UNITS. True means they are requested in integer subtoken units.
+* `BNDL` (default: `False`) means WE DO NOT ALLOW BUNDLE PRICING ON NEW ORDERS, ONLY INTEGER PRICES. True means we allow Fraction prices.
+* `STRICT` (default: `False`) means WE CREATE OR EXECUTE NEAREST VALID TRADE <= REQUESTED TRADE. True means invalid trades throw an error.
+* `DEBUG` (default: `False`) enables logs to console
+* `TESTMODE` (default: `False`) enables testmode, forcedisable send transaction
+* These flags can and should be ignored and left out of function calls unless the user wants to change them.
+
+### Depencies
+* python3+ https://www.python.org/downloads/
 * web3
 * eth_account
 * ethereum
 * pandas
 * base58
 * ecdsa
+
+### Install
+* `pip3 install web3, eth_account, ethereum, pandas, base58, ecdsa`
+
+### Sample Usage
+> NOTE: The biggest difference to saturn.js is, that you have to create seperate instances if you want to use ETH and ETC!
+
+~~~py
+#!/usr/bin/env python3
+from saturnpy import Saturn
+
+# EDIT / LOAD etcprofile.json or create settings-obj in file.
+
+settings = {
+    "blockchain": "ETC",
+    "apiUrl": "https://ticker.saturn.network/api/v2/",
+    "providerUrl": "https://etc-rpc.binancechain.io",
+    "mnemonicKey": "legal winner thank year wave sausage worth useful legal winner thank yellow",
+    "testMode": True,
+    "DEBUG": True
+}
+
+#you can also add privateKey to settings
+settingsETH = {
+    "blockchain": "ETH",
+    "apiUrl": "https://ticker.saturn.network/api/v2/",
+    "providerUrl": "http://localhost:8545",
+    "privateKey": "0xKeyToShuttle",
+    "testMode": False,
+    "DEBUG": False
+}
+
+# Saturn instances
+etc = Saturn(settings)
+eth = Saturn(settingsETH)
```

Comment:
Was debating whether it is better to do this way, or the way saturn.js currently does this. We'll see if in saturn.js 2.0 will adopt this style